### PR TITLE
login screen - api integration

### PR DESCRIPTION
**Pull request by dawoodkhan4050**
- login screen code refactor
- login screen api integration(happy scenarios)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -35,6 +37,8 @@ function SihWizard (){
         <ContentWrapper>
             <Routes>
                 <Route path="/" element={ <Welcome/> } />
+                <Route path="/login" element={ <LoginForm/> } />
+                <Route path="/wizard" element={ <Authenticate> <CarePlanWizard /> </Authenticate> } />
                 <Route path="/settings" element={ <Authenticate> <Settings /> </Authenticate> } />
```
Comment (faheemsajjad65) [inline src/App.js]: @hawaism no need to add login and wizard route

File: src/views/Welcome.js [hunk 2]
```diff
@@ -1,11 +1,8 @@
 import React from 'react';
-import LoginForm from '../components/LoginForm';
-import CarePlanWizard from "./CarePlanWizard"
-
 
 export default function Welcome() {
     
     return (
-        false ? <LoginForm /> : <CarePlanWizard />
+        <div>welcome page</div>
```
Comment (faheemsajjad65) [inline src/views/Welcome.js]: @hawaism please undo all these

File: src/reducers/auth.js [hunk 3]
```diff
@@ -32,6 +35,18 @@ export default function (state = initialState, action) {
                 isLoggedIn: false,
                 user: null,
             };
+        case LOGIN_ATTEMPT_INIT:
```
Comment (faheemsajjad65) [inline src/reducers/auth.js]: @hawaism no need to add this case as it automatically redirects to default.

File: src/App.js (hunk 1 above)
Comment (dawoodkhan4050) [reply src/App.js]: resolved

File: src/views/Welcome.js (hunk 2 above)
Comment (dawoodkhan4050) [reply src/views/Welcome.js]: resolved

File: src/reducers/auth.js (hunk 3 above)
Comment (dawoodkhan4050) [reply src/reducers/auth.js]: done

**Review by faheemsajjad65 — APPROVED** [review]

**Pull request merged**